tests/networking.unit.tftest.hcl

```diff
@@ -0,0 +1,269 @@
+mock_provider "azurerm" {}
+
+run "setup_tests" {
+  module {
+    source = "./tests/setup"
+  }
+}
+
+run "basic_sku" {
+  command = plan
+
+  variables {
+    namespace_name             = run.setup_tests.namespace_name
+    resource_group_name        = run.setup_tests.resource_group_name
+    location                   = run.setup_tests.location
+    log_analytics_workspace_id = run.setup_tests.log_analytics_workspace_id
+  }
+
+  assert {
+    condition     = length(azurerm_servicebus_namespace.this.network_rule_set) == 0
+    error_message = "Network rule set block created when it should not have been"
+  }
+}
```

Comment:
Am I understanding it correctly that we are we checking here if a network rule set block has been created, despite the `network_rule_set` list is empty?

Reply:
Since its beneath the `basic_sku` test, is `network_rule_set` not availiable for basic SKU?

Reply:
Same for the `standard_sku` test.

Reply:
Correct, network rules can only be configured for premium SKU, so for the basic and standard SKUs we only need to do a single check: verify that the network rule set block has **not** been created.

Reply:
Alright 👍🏼